features/NodeProcessor/TextNodeProcess.php

```diff
@@ -0,0 +1,28 @@
+<?php
+declare(strict_types=1);
+
+namespace Netlogix\XmlProcessor\Behat\NodeProcessor;
+
+use Netlogix\XmlProcessor\NodeProcessor\AbstractNodeProcessor;
+use Netlogix\XmlProcessor\NodeProcessor\Context\OpenContext;
+use Netlogix\XmlProcessor\NodeProcessor\Context\TextContext;
+use Netlogix\XmlProcessor\NodeProcessor\OpenNodeProcessorInterface;
+use Netlogix\XmlProcessor\NodeProcessor\TextNodeProcessorInterface;
+
+class TextNodeProcess extends AbstractNodeProcessor implements TextNodeProcessorInterface, InvokeNodeProcessorInterface
+{
+    public const NODE_PATH = 'test';
```

Comment:
Ist 'test' hier korrekt?

Reply:
Ja, ich teste hier das **NodeType Event Text**.
Der `TextNodeProcess` sammelt die Texte in allen test xml nodes.

Reply:
Ahh ja das ist für den Behat Test. Ich dachte es wäre eine ganz normale Klasse 😅